web/src/components/Game/ticker.js

```diff
@@ -0,0 +1,39 @@
+// Provides smooth interpolated X position based on timekeeping "ticks" from the server
+
+import { useState, useMemo, useRef, useEffect } from 'react';
+
+import StreamingInterpolator from 'helpers/streaming-interpolation';
+
+import { MOVING_SPEED } from './constants';
+const TICKS_PER_SECOND = 8;
```

Comment:
Re note above, since this is already a constant here, consider moving this to constants.js and importing from there